src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -1188,6 +1199,19 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  private adjustCoordinates(): void {
+    const timelineActionsBandWidth = 404;
```

Comment:
It depends on the language due to the translation of 'From' and 'To', as well as whether the timeline has a selection

Reply:
Fixed, I took the max value